pkg/network/netlink/align.go

```diff
@@ -0,0 +1,42 @@
+package netlink
+
+import (
+	"unsafe"
+
+	"github.com/mdlayher/netlink"
+)
+
+// Copied from github.com/mdlayher/netlink/align.go
+// Functions and values used to properly align netlink messages, headers,
+// and attributes. Definitions taken from Linux kernel source.
+
+// #define NLMSG_ALIGNTO 4U
+const nlmsgAlignTo = 4
+
+// #define NLMSG_ALIGN(len) ( ((len)+NLMSG_ALIGNTO-1) & ~(NLMSG_ALIGNTO-1) )
+func nlmsgAlign(len int) int {
+	return ((len) + nlmsgAlignTo - 1) & ^(nlmsgAlignTo - 1)
```

Comment:
Curious - what does the `()` around `len` do?

Reply:
This file is a copy-paste from the the netlink library which is in turn is a copy-paste from the kernel source code so 🤷‍♂️ 
I think in that case the parenthesis might help to disambiguate the `len` variable name with the `len` built-in function.